Client/Application/AppDelegate.swift

```diff
@@ -387,6 +388,33 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         SDWebImageDownloader.shared.setValue("image/*;q=0.8", forHTTPHeaderField: "Accept")
     }
 
+    fileprivate func askForReview() {
+        guard self.shouldAskForReview() else {
+            return
+        }
+        SKStoreReviewController.requestReview()
+    }
+
+    private func shouldAskForReview() -> Bool {
+        let dateformat = DateFormatter()
+        dateformat.dateFormat = "yyyyMMdd"
+        let today = dateformat.string(from: Date())
+        guard UserAgentConstants.installDate != today else {
+            self.profile?.prefs.setBool(false, forKey: PrefsKeys.ShowAppReview)
+            return false
+        }
+        let showAppReview = self.profile?.prefs.boolForKey(PrefsKeys.ShowAppReview)
+        guard showAppReview == nil || showAppReview! else {
+            return false
+        }
+        self.profile?.prefs.setBool(false, forKey: PrefsKeys.ShowAppReview)
+        let random = Int.random(in: 1...100)
```

Comment:
Lets ask for review everytime if in debug, okay?

Reply:
Done